lib/pushy_client.rb

```diff
@@ -206,7 +206,10 @@ def on_job_state_change(&block)
   private
 
   def rest
-    @rest ||= Chef::REST.new(chef_server_url, client_name, client_key)
+    @rest ||= begin
+      require 'chef/rest'
+      Chef::REST.new(chef_server_url, client_name, client_key)
+    end
```

Comment:
This change was yanked from https://github.com/chef/opscode-pushy-client/pull/84. I actually don't think this commit should be included with the PR and instead opt for resolving this particular issue from https://github.com/chef/opscode-pushy-client/pull/84